webviewer/src/pages/Signup/components/Card/ConfirmCard.jsx

```diff
@@ -3,9 +3,16 @@ import { Button } from 'components/_shared/Button';
 import { Header, CardText, CardWrap } from './Card.styles';
 import { useHistory } from 'react-router-dom';
 import { DEFAULT_ERROR, ROUTES } from '_constants';
+import { useUserActions } from 'state';
 
 export const ConfirmCard = ({ error }) => {
   const history = useHistory();
+  const { setAutoLogged } = useUserActions();
+
+  const handleContinueClick = async () => {
+    setAutoLogged(true);
```

Comment:
todo: please think on in future about change function name. because it's implementation has some concern with it's naming